intercom_flutter_web/lib/intercom_flutter_web.dart

```diff
@@ -155,10 +154,49 @@ class IntercomFlutterWeb extends IntercomFlutterPlatform {
       userAttributes['user_id'] = userId;
     }
 
-    if (company != null && companyId != null) {
+    if (company != null) {
       Map<String, dynamic> companyObj = {};
-      companyObj['company_id'] = companyId;
-      companyObj['name'] = company;
+
+      if (company.containsKey("company_id")) {
+        companyObj['company_id'] = company['company_id'] as String;
+      }
+
+      if (company.containsKey("name")) {
+        companyObj['name'] = company['name'] as String;
+      }
+
+      if (company.containsKey("created_at")) {
+        companyObj['created_at'] = company['created_at'] as String;
+      }
+
+      if (company.containsKey("plan")) {
+        companyObj['plan'] = company['plan'] as String;
+      }
+
+      if (company.containsKey("monthly_spend")) {
+        companyObj['monthly_spend'] = company['monthly_spend'] as String;
```

Comment:
The same here. Looks like it is `int` type

Reply:
done, thanks man!